controller/src/main/java/com/controller/collectors/MySQLCollector.java

```diff
@@ -0,0 +1,98 @@
+package com.controller.collectors;
+
+import com.controller.util.JSONUtil;
+import com.controller.util.json.JSONException;
+import com.controller.util.json.JSONObject;
+import com.controller.util.json.JSONStringer;
+
+import java.sql.*;
+
+/**
+ *
+ */
+public class MySQLCollector extends DBCollector {
+//    private static final Logger LOG = Logger.getLogger(MySQLCollector.class);
```

Comment:
Delete all commented out code

Reply:
I left the commented code there because in future we might want to add back the logging function even though it is not the focus for now. Should we still remove it?

Reply:
Would it be much work to set up log4j?

Reply:
Just set up lo4j.